runtime/cudaq/platform/orca/OrcaExecutor.h

```diff
@@ -17,6 +17,10 @@ namespace cudaq {
 /// API.
 class OrcaExecutor : public Executor {
 public:
+  details::future
+  execute(std::vector<KernelExecution> &codesToExecute) override {
+    throw std::runtime_error("execute on kernels not supported");
```

Comment:
```suggestion
    throw std::runtime_error("ORCA backend does not support executing arbitrary kernels");
```
This message could be user facing. I am hence looking for something a bit more descriptive. @khalatepradnya please confirm that the suggested edit is accurate.